### PR TITLE
Docs: Create basic plugin guide improvements

### PR DESCRIPTION
**Pull request by dufipl**
Multiple improvements in the basic plugin guide including more detailed instructions for the DIY version.

### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs: Create basic plugin guide improvements. Closes https://github.com/cksource/ckeditor5-internal/issues/2191.

### CODE REVIEW

File: docs/framework/guides/creating-simple-plugin-timestamp.md [hunk 1]
```diff
@@ -180,7 +215,12 @@ class Timestamp extends Plugin {
 		const editor = this.editor;
 
 		editor.ui.componentFactory.add( 'timestamp', () => {
-			//...
+			const button = new ButtonView();
+
+            button.set( {
+                label: 'Timestamp',
+                withText: true
+            } );
```
Comment (Mgsy) [inline docs/framework/guides/creating-simple-plugin-timestamp.md:223]: See my previous comment. I'm not sure if in this case adding the full snippet is a good idea, as I have a feeling of breaking the flow and distracting - "should I change something in my `ButtonView` definition too?". If we have `// ...` for the unchanged part of the code, we make it clear that now we work on a new part of the code and the previous one remains unchanged.

Comment (dufipl) [reply docs/framework/guides/creating-simple-plugin-timestamp.md]: It was fixed as agreed by using an ellipse with the comment.

File: docs/framework/guides/creating-simple-plugin-timestamp.md [hunk 2]
```diff
@@ -118,7 +145,15 @@ Once we create a new instance of the `ButtonView`, we will be able to customize
...
+import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
+import Essentials from '@ckeditor/ckeditor5-essentials/src/essentials';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import Heading from '@ckeditor/ckeditor5-heading/src/heading';
+import List from '@ckeditor/ckeditor5-list/src/list';
+import Bold from '@ckeditor/ckeditor5-basic-styles/src/bold';
+import Italic from '@ckeditor/ckeditor5-basic-styles/src/italic';
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
 import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
```
[the remark this answers is not in the document]
Comment (dufipl) [reply docs/framework/guides/creating-simple-plugin-timestamp.md]: It was fixed as agreed by using an ellipse with the comment.

**Comment by dufipl** [comment]
@Mgsy I've pushed the fix according to the review and agreement made on the CCX sync, could you take another look? :)

**Review by Mgsy — APPROVED** [review]

**Pull request merged**